src/renderer/App.css

```diff
@@ -30,23 +30,23 @@
 .flexBox {
   display: block;
   user-select: unset;
-  -webkit-user-select: unset;
 }
 
-#changeLogText {
+.changeLogText {
   overflow-y: scroll;
   block-size: 40vh;
-  display: block
+  display: block;
 }
 
 .fade-enter-active, .fade-leave-active {
   transition: opacity .15s;
 }
+
 .fade-enter, .fade-leave-to /* .fade-leave-active below version 2.1.8 */ {
   opacity: 0;
 }
 
-@media only screen and (max-width: 680px) {
+@media only screen and (width <= 680px) {
```

Comment:
This works coz we use electron = Chrome
https://caniuse.com/css-media-range-syntax

Reply:
It does work, but it's not the opinion of [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard). See [here](https://stylelint.io/user-guide/rules/media-feature-range-notation/) and [here](https://github.com/stylelint/stylelint-config-standard/blob/9d8d64daf469c6e7466594ceb003065556bcecba/index.js#L83)

Reply:
Will this newer syntax break the media queries for FreeTubeCordova? That's where most of the users of the media queries will be.
Looks like you can enforce the old style instead of the new style by setting that stylelint setting to the other value.

Reply:
Looks like they're using [Electron v22](https://github.com/MarmadileManteater/FreeTubeCordova/blob/development/package.json), [so we should be good](https://www.electronjs.org/blog/electron-22-0)

Reply:
The linter is not smart enough to fix this rule violation the other way apparently... will have to do some better research to answer that question

Reply:
Based on Emma's response in the channel, I think we're good?